alacritty/src/renderer/mod.rs

```diff
@@ -535,7 +544,7 @@ impl Batch {
 }
 
 /// Maximum items to be drawn in a batch.
-const BATCH_MAX: usize = 0x1_0000;
+const BATCH_MAX: usize = 0x1000;
```

Comment:
Why the batch size was changed, please revert it back?

Reply:
One batch takes around 4.3M on my setup, so these changes will increase global memory consumption from that amount. I figured that BATCH_MAX was quite big but I then realized that it's probably designed to hold a full screen at once.

I'll change it back.

Reply:
The thing is that lower batch size will result in massive flickering all over the place, and 4MiB isn't something that bad.